db/syncPostsToGrapher.ts

```diff
@@ -222,6 +299,24 @@ const syncPostsToGrapher = async (): Promise<void> => {
             featured_image: post.featured_image || "",
         }
     }) as PostRow[]
+    const postLinks = await dataSource.getRepository(PostLink).find()
```

Comment:
Don't know if there's a difference somewhere but this should also work?

```suggestion
    const postLinks = await PostLink.find()
```